vSMR/SMRPlugin.cpp

```diff
@@ -734,7 +740,7 @@ void CSMRPlugin::OnTimer(int Counter)
 	else if (!AutoConnect && HoppieConnected)
 	{
 		HoppieConnected = false;
-		DisplayUserMessage("CPDLC", "Server", "Logged off!", true, true, false, true, false);
+		DisplayUserMessage("CPDLC", "Server", "Please run .smr connect to login!", true, true, false, true, false);
 	}

```

Comment:
I ought to have made my previous comments clearer; sorry. This entire block makes little sense, since the effect is to, for any controller with the auto-connect feature disabled when connected to Hoppie's, every second, disconnect them from Hoppie's and inform them to connect. The `else` is also redundant; the conditions are exclusive.